_config.yml

```diff
@@ -24,8 +24,8 @@ description: >- # this means to ignore newlines until "baseurl:"
   The Kani Rust Verifier (Kani) is an open-source automated reasoning
   tool that proves properties about Rust code. This is our blog for
   the project.
-baseurl: "kani-verifier-blog/" # the subpath of your site, e.g. /blog
-url: "" # the base hostname & protocol for your site, e.g. http://example.com
+baseurl: "kani-verifier-blog" # the subpath of your site, e.g. /blog
+url: "https://model-checking.github.io/" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
I don't know the extent to which this matters, but at least one thing I was seeing suggests it might matter somehow where the slash is.

`/kani-verifier-blog`

and

`"https://model-checking.github.io"`